src/MainUpdate.elm

```diff
@@ -331,7 +331,10 @@ update msg model =
                 ( (updateNewSession model (\ns -> { ns | location = newLocation })), Cmd.none )
 
             UpdateNewSessionTrack newTrackId ->
-                ( (updateNewSession model (\ns -> { ns | trackId = (toInt model newTrackId) })), Cmd.none )
+                if newTrackId == "" then
+                    ( (updateNewSession model (\ns -> { ns | trackId = Nothing })), Cmd.none )
```

Comment:
I dont think we need double parens here or below.

Also we should try to use types that match their possible values.  If `newTrackId` can only be an integer or "null" then we should use an `Maybe Int` type